rllib/env/external_env.py

```diff
@@ -218,15 +217,15 @@ def log_action(self, observation, action):
             self.new_observation = observation
             self.new_action = action
         self._send()
-        self.action_queue.get(True, timeout=60.0)
+        self.action_queue.get(True, timeout=1000000.0)
 
     def wait_for_action(self, observation):
         if self.multiagent:
             self.new_observation_dict = observation
         else:
             self.new_observation = observation
         self._send()
-        return self.action_queue.get(True, timeout=60.0)
+        return self.action_queue.get(True, timeout=1000000.0)
```

Comment:
This isn't an appropriate default timeout, it will lead to indefinite hangs on error, can you set it back to 60.0? If it needs to be configurable, we can make it an option in the constructor or an internal class attribute.

Reply:
Good catch.

Reply:
done